### PR TITLE
Update LICENSE.txt

### PR DESCRIPTION
**Pull request by alex4200**
Update of the LICENSE file according to requirement.

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,24 +1,9 @@
 BluePyOpt - Bluebrain Python Optimisation Library
 
-BluePyOpt is licensed under the LGPL, unless noted otherwise, e.g., for external 
-dependencies. See file LGPL.txt for the full license. 
-Examples and test are BSD-licensed.
```
Comment (wvangeit) [inline LICENSE.txt:5]: What happened with special case of examples/tests?

Comment (alex4200) [reply LICENSE.txt:5]: What do you mean?

Comment (jdcourcol) [reply LICENSE.txt:5]: The examples and tests have a different license (BSD) than the source code (LGPL)
The line stating that is removed with your changes.

Comment (alex4200) [reply LICENSE.txt:5]: Then please disregard my changes. I was not aware a repo can have multiple licenses. Not sure how to handle those.

Comment (wvangeit) [reply LICENSE.txt:5]: I leave this PR open so you can make more changes? Or do I close it for now?

Comment (alex4200) [reply LICENSE.txt:5]: Or maybe you can leave it open.

Comment (wvangeit) [reply LICENSE.txt:5]: Ok

**Comment by wvangeit** [comment]
Does this have to stay open ?

**Pull request closed without merging**